src/CabanaPD_Types.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace CabanaPD
 {
+// Fracture types.
 struct Elastic
 {
 };
 struct Fracture
 {
 };
+
+// Thermal types.
 struct TemperatureDependent
 {
 };
 struct TemperatureIndependent
 {
 };
+struct DynamicTemperature : public TemperatureDependent
```

Comment:
So, I guess we have temperature independent/dependent mechanics and, in addition, dynamic temperature, i.e., inclusion of heat transfer solver?

What happens if a user defines a temperature independent model with "dynamic temperature"? This would be a suitable option is someone wants to solve a pure (non-mechanical) heat transfer problem.

Reply:
That's not currently possible and I don't think it's necessarily worth the effort

Reply:
I see. So, solving heat transfer assumes temperature dependent mechanics?

I believe an option for solving pure heat transfer could be to solve the coupled system with zero coefficient of linear thermal expansion.

Reply:
It's a matter of what's actually implemented - you could write a pure heat transfer problem with this code, but there's no current solver that would make that as simple as all the current examples